jenkins_jobs/gpii-app.yml

```diff
@@ -61,6 +67,16 @@
           # Mark the build as failed
           fail: true
 
+- job:
```

Comment:
Perhaps this is the desired behavior, but should it have a timeout and a failure of the job if the command fails as the other jobs?:

https://github.com/GPII/ci-service/blob/e1e819c2dde819bfa5cefd49ff1842becda859a8/jenkins_jobs/gpii-app.yml#L63-L68

Reply:
Fair enough, I just added the timeout as requested.